_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Google Assistant Development for Android Devs (Part 1)"
+author: Patrick Jackson
+categories: Development
+tags: [Google Assistant, Actions on Google, conversational interfaces, Android, Kotlin]
+---
+
+
+# Google Assistant Development for Android Devs (Part 1)
+
+![](https://storage.googleapis.com/banjotms.appspot.com/assistant_surfaces.svg)
+
+There is a lot of buzz and excitement around the [Google Assistant Actions](https://developers.google.com/actions/), and with good reason:  this is an entirely new platform for building experiences for multiple devices.  Building a conversational Action with Google Assistant means it will be available on Android, iOS, Google Home, and others as they become available.
+
```

Comment:
Add a read-more break here:
`<!--more-->`

Otherwise, the image will be the only thing that shows in the excerpt.